### PR TITLE
added ruby 2.6 and jruby to test matrix

### LINKED ISSUE #140: Add JRuby to the testing matrix for travis
JRuby should be tested via Travis. Rubinus wouldn't be a bad idea either.

### PR DESCRIPTION
**Pull request by walf443**
ruby 2.6.x was already released
https://www.ruby-lang.org/en/news/2019/01/30/ruby-2-6-1-released/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,4 +12,5 @@ rvm:
   - 2.3
   - 2.4
   - 2.5
-
+  - 2.6
```
Comment (grosser) [inline .travis.yml]: might have to use 2.6.1 or 2.6.0 ... not sure why it does not work ... or google about supported travis ruby versions :/

Comment (grosser) [reply .travis.yml]: ... maybe remove the dist+sudo parts ...

Comment (walf443) [reply .travis.yml]: Thanks quick response. It may be that precise dist is too old.

Comment (walf443) [reply .travis.yml]: I see. it should consider http://rubies.travis-ci.org/

**Comment by walf443** [comment]
I'll fix this deprecation warning failure at later time https://travis-ci.org/grosser/parallel/jobs/495262000

**Comment by grosser** [comment]
can either bump activesupport or set it to ignore deprecations 🤷‍♂️

**Comment by walf443** [comment]
I give up to add jruby on this matrix.
It maybe better to update travis environment to Ubuntu 16.04 before this.

File: spec/parallel_spec.rb [hunk 2]
```diff
@@ -188,7 +188,7 @@ def cpus
 
     it 'does not leave processes behind while running' do
       skip if ENV['TRAVIS'] # this randomly fails on travis all the time :(
-      `ruby spec/cases/closes_processes_at_runtime.rb`.should == 'OK'
+      `ruby spec/cases/closes_processes_at_runtime.rb`.split(/\n/).last.should == 'OK'
```
Comment (grosser) [inline spec/parallel_spec.rb]: this is a jruby fix ... but jruby got removed ?

Comment (walf443) [reply spec/parallel_spec.rb]: It's for ruby 2.6's BigDecimal warnining https://travis-ci.org/grosser/parallel/jobs/495262000

**Comment by grosser** [comment]
let's do that then :)

**Comment by walf443** [comment]
@grosser I'm sorry to fixed it lately. All test passed.

File: spec/parallel_spec.rb (hunk 2 above)
Comment (grosser) [reply spec/parallel_spec.rb]: would prefer to ignore that explicitly then
`.sub(/.*use BigDecimal.*\n/, "")`

**Comment by grosser** [comment]
thx for trying, every little step helps :)

File: spec/parallel_spec.rb (hunk 2 above)
Comment (walf443) [reply spec/parallel_spec.rb]: fixed

File: spec/parallel_spec.rb [hunk 3]
```diff
@@ -188,7 +188,7 @@ def cpus
 
     it 'does not leave processes behind while running' do
       skip if ENV['TRAVIS'] # this randomly fails on travis all the time :(
-      `ruby spec/cases/closes_processes_at_runtime.rb`.should == 'OK'
+      `ruby spec/cases/closes_processes_at_runtime.rb`.gsub(/[^\n]* deprecated; use BigDecimal[^\n]*\n/, '').should == 'OK'
```
Comment (grosser) [inline spec/parallel_spec.rb]: no need for `[^\n]` since regexes don't jump lines

`"abc\nb".gsub(/a.*/, "") == "\nb"`

**Pull request merged**